### ISSUE TITLE
Standalone Graphics

### ISSUE THREAD

**Issue by oliver----** [substance/texture#712]
In JATS it is valid to have graphic as a block-element, i.e. not necessarily inside a figure.
This concept makes sense in general, but maybe we would approach in a different way, e.g. for UX reason.
Such a standalone graphic could be seen as a figure with an image but without title, caption, and label.

**Comment by oliver----** [comment]
What I am referring to is mentioned in https://jats.nlm.nih.gov/archiving/tag-library/1.1/element/graphic.html 'Best Practises`, a figure without the boiler-plate.

**Comment by oliver----** [comment]
And what I am suggesting is that UX wise it is easier to grasp that there are figures with or without captions, instead of introducing two concepts. 
Regarding JATS I would probably follow the mentioned best-practice, and tag a figure without caption as `<graphic>`

**Comment by michael** [comment]
Also for treating block images and block figures as Figures internally. However for inline graphics we need a separate concept. Maybe put the button next to the annotations (strong, emphasis, math)?

Anyway let's put this on the agenda for the Sao Paulo meet. Assigning 2.0 Milestone.

**Comment by oliver----** [comment]
I guess an 'inline-graphic' is a too difficult concept for representing it just as an icon. So I would put these 'advanced' content types into a menu. Similar case is `disp-formula` and `inline-formula`.

**Comment by michael** [comment]
Maybe we should ask the user on insertion. Say cursor is inside a paragraph and you click the figure tool. Then we could prompt for "Want to create a figure or a graphic inside the text flow?". And then either break the para with a block node or insert the inline graphic. To be discussed in Sao Paulo.

**Comment by oliver----** [comment]
UX wise this is less important for Sao Paola. It is more for the authoring use-case.

**Comment by michael** [comment]
Yeah true. I guess from a JATS view it's all clear what to do here (graphic elements as blocks and inline). And it's on us to provide good UX.

**Comment by oliver----** [comment]
Best UX for start is Menu.
